slapper.go

```diff
@@ -283,7 +284,8 @@ func reporter(quit <-chan struct{}) {
 			fmt.Print("\033[H") // clean screen
 			fmt.Printf("sent: %-6d ", sent)
 			fmt.Printf("in-flight: %-2d ", sent-recv)
-			fmt.Printf("\033[96mrate: %4d RPS\033[0m ", currentRate.Load())
+			fmt.Printf("requested rate: %4d RPS ", requestedRate.Load())
+			fmt.Printf("\033[96mresponse rate: %4d RPS\033[0m ", currentRate.Load())
```

Comment:
I followed your suggestion here and renamed requestedRate to 'desiredRate' and changed the status line to "rate %4d/%d RPS".